lti_consumer/plugin/urls.py

```diff
@@ -16,12 +16,15 @@
     LtiAgsLineItemViewset,
     deep_linking_response_endpoint,
     deep_linking_content_endpoint,
+    # LTI NRPS URLs
+    LtiNrpsContextMembershipViewSet,
 )
 
 
 # LTI 1.3 APIs router
 router = routers.SimpleRouter(trailing_slash=False)
 router.register(r'lti-ags', LtiAgsLineItemViewset, basename='lti-ags-view')
+router.register(r'lti-nrps/memberships', LtiNrpsContextMembershipViewSet, basename='lti-nrps-memberships-view')
```

Comment:
@shimulch You can simplify this to just `memberships` :)
```suggestion
router.register(r'memberships', LtiNrpsContextMembershipViewSet, basename='lti-nrps-memberships-view')
```